components/job-orchestration/job_orchestration/executor/compress/compression_task.py

```
@@ -68,19 +70,20 @@ def increment_compression_job_metadata(db_cursor, job_id, kv):
     db_cursor.execute(query)
 
 
-def update_tags(db_cursor, table_prefix, archive_id, tag_ids):
+def update_tags(db_cursor, table_prefix, dataset, archive_id, tag_ids):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)